icubam/backoffice/server_test.py

```diff
@@ -90,3 +90,32 @@ def test_consent_reset(self):
     self.assertIn('error', resp_data)
     self.assertIn('msg', resp_data)
     self.assertIsNone(resp_data['error'])
+
+  def test_upload(self):
+    handler = upload.UploadHandler
+    with mock.patch.object(base.BaseHandler, 'get_current_user') as m:
+      m.return_value = self.admin
+      data_icus = dict()
+      with open("resources/test/icu2.csv") as csv_f:
+        data_icus['data'] = csv_f.read()
+        data_icus['objtype'] = base.ObjType.ICUS.value
```

Comment:
Using Enum internally sounds good, but for API users I think it's still nicer and more expressive to pass a string as a argument instead of int (and checking out the mapping table). For instance here we could have used enum name instad.